sdk/ai/azure-ai-agents/azure/ai/agents/models/_patch.py

```diff
@@ -854,8 +854,10 @@ def __init__(self, connection_id: str, market: str = "", set_lang: str = "", cou
         :param market: The market where the results come from.
         :param set_lang: The language to use for user interface strings when calling Bing API.
         :param count: The number of search results to return in the Bing API response.
-        :param freshness: Filter search results by a specific time range. Accepted values: 
-           https://learn.microsoft.com/bing/search-apis/bing-web-search/reference/query-parameters
+        :param freshness: Filter search results by a specific time range.
+        
+        .. seealso:: 
+           `Bing Web Search API Query Parameters <https://learn.microsoft.com/bing/search-apis/bing-web-search/reference/query-parameters>`_
         """
         self.connection_ids = [
```

Comment:
we might want to rename self.connection_ids to internal variable, e.g. self._search_configurations, because customer shall not access them directly but via definitions.